src/catalog/src/expr_cache.rs

```diff
@@ -153,41 +165,38 @@ impl ExpressionCache {
             durable_cache,
         };
 
-        while let Err(err) = cache
-            .try_open(current_ids, optimizer_features, remove_prior_gens)
-            .await
-        {
-            debug!("failed to open cache: {err} ... retrying");
+        loop {
```

Comment:
We should probably have a max number of retires or a sleep in-between attempts here? I wouldn't want a bug to DDOS persist

Reply:
Done